src/software/ai/motion_constraint/motion_constraint_visitor.cpp

```diff
@@ -84,7 +84,13 @@ void MotionConstraintVisitor::visit(const DribbleTactic &tactic) {}
 
 void MotionConstraintVisitor::visit(const GetBehindBallTactic &tactic) {}
 
-void MotionConstraintVisitor::visit(const MoveToGoalLineTactic &tactic) {}
+void MotionConstraintVisitor::visit(const MoveToGoalLineTactic &tactic)
+{
+    current_allowed_constraints = std::set<MotionConstraint>({
```

Comment:
add a comment that movetogoallinetactic moves to goal line in any gamestate

Reply:
done